aws/resource_aws_eks_cluster_test.go

```diff
@@ -171,6 +173,98 @@ func TestAccAWSEksCluster_VpcConfig_SecurityGroupIds(t *testing.T) {
 	})
 }
 
+func TestAccAWSEksCluster_VpcConfig_EndpointPrivateAccess(t *testing.T) {
+	var cluster1, cluster2, cluster3 eks.Cluster
+
+	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	resourceName := "aws_eks_cluster.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSEksClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSEksClusterConfig_VpcConfig_EndpointPrivateAccess(rName, true),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSEksClusterExists(resourceName, &cluster1),
+					resource.TestCheckResourceAttr(resourceName, "vpc.config.#", "1"),
```

Comment:
Urg! Sorry I left a typo in there. All these should be `vpc_config.` not `vpc.config.` if you want to fix this 😅 

Other than my mistake here, I think this is ready to go all the other testing is passing for me and I'm running the type fixed testing now (which I'm expecting to be just fine).

Reply:
I'll fix; Push in a minute.

Reply:
Pushed fix.

Reply:
💯 🥇